less/misc.less

```diff
@@ -53,6 +53,9 @@ input, textarea {
 	visibility: hidden;
 	top: 0;
 	float: left;
+	.selectlist {
+		display: block !important;
```

Comment:
lets provide a more explicit class name then to make it clear it's a dummy
`.dummy-selectlist`